# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../brscan/sanify.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- brscan-src-0.2.4/brscan/backend_src/brother_modelinf.c.orig	2008-02-16 12:41:06.000000000 +0100
+++ brscan-src-0.2.4/brscan/backend_src/brother_modelinf.c	2008-02-16 12:44:19.000000000 +0100
@@ -94,15 +94,15 @@
 static int modelListGetEnable = FALSE;
 
 
-static const char bwString[]            = "Black & White";
-static const char errDiffusionString[]  = "Gray[Error Diffusion]";
-static const char tGrayString[]         = "True Gray";
-static const char ColorString[]       = "24bit Color";
-static const char ColorFastString[]   = "24bit Color[Fast]";
-
-static const char fbString[] = "FlatBed";
-static const char adfString[] = "Automatic Document Feeder";
-static const char adfdupString[] = "Automatic Document Feeder(Duplex)";  //06/02/27
+static const char bwString[]            = "Line";
+static const char errDiffusionString[]  = "Dithered";
+static const char tGrayString[]         = "Gray";
+static const char ColorString[]       = "Color";
+static const char ColorFastString[]   = "Color[Fast]";
+
+static const char fbString[] = "Flatbed";
+static const char adfString[] = "ADF";
+static const char adfdupString[] = "ADF Duplex";  //06/02/27
 
 #define MAX_PATH 256
 
